### PR TITLE
Download photos, photos UI

### PR DESCRIPTION
**Pull request by ServingCoder**
Add the ability to upload photos when attendees view events. I also changed a little bit of UI so different photo aspect ratios can exist and still look nice.

### CODE REVIEW

File: code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfo.java [hunk 1]
```diff
@@ -95,26 +99,45 @@ private void baseSetup(String eventId){
...
-            //Log.d("EventInfo", event.getName());
-            //Log.d("EventInfo", event.getDetails());
-            //Log.d("EventInfo", event.getLocation());
-            //organizerText.setText(); // TODO: update organizer (not implemented in event yet)
-            // TODO: update organizer image
-            //eventLocation.setText(); // TODO: format LocalDateTime with current API lvl
-            // TODO: update event image
+        task.addOnSuccessListener(new OnSuccessListener<DocumentSnapshot>() {
+            @Override
+            public void onSuccess(DocumentSnapshot documentSnapshot) {
```
Comment (WhoShotCupidd) [inline code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfo.java:104]: add javadoc

File: code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfo.java [hunk 2]
```diff
@@ -95,26 +99,45 @@ private void baseSetup(String eventId){
...
+
+                //download the event image from db and populate the screen
+                eventImage = findViewById(R.id.eventImage);
+                String findImage = "event_banners/"+event.getId()+"-upload";
+                // set imageview and update organizer image preview
+                if (FirebaseStorage.getInstance().getReference().child(findImage) != null) {
+                    MainActivity.db.downloadPhoto(findImage,
+                            t -> runOnUiThread(() -> eventImage.setImageBitmap(t)));
+                }
+                //Note for when we download organizer photo:
```
Comment (WhoShotCupidd) [inline code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfo.java:126]: these comments still needed?

File: code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfo.java (hunk 1 above)
Comment (ServingCoder) [reply code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfo.java:104]: I didn't write this part.

**Review by WhoShotCupidd — APPROVED** [review]
lgtm

**Pull request merged**